core/src/main/java/org/apache/calcite/ExceptionLoggingAspect.java

```diff
@@ -61,4 +61,12 @@ public List<String> getSqlExpressions() {
   public List<String> getRelExpressions() {
     return details.get(REL_EXPRESSION);
   }
+
+  public void clear() {
+    details.get(REL_NODE).clear();
+    details.get(METHOD_CALL).clear();
```

Comment:
public void clear() {
    details.clear();
}
details.clear() is sufficient because

details is a Map<String, List<String>>, and calling .clear() removes all keys and their associated lists.

or (without removing the keys)
public void clear() {
    details.values().forEach(List::clear);
}

can we change this

Reply:
done

Reply:
done